.circleci/config.yml

```diff
@@ -1,6 +1,54 @@
 version: 2
 
 jobs:
+  # test-ruby-2.5:
+  #   docker:
+  #   - image: circleci/ruby:2.5.1-stretch
+  #   working_directory: ~/googleapis/google-cloud-ruby
+  #   shell: /bin/bash --login -eo pipefail
+  #   steps:
+  #   - checkout
+  #   - run:
+  #       name: Install bundle
+  #       command: gem update --system && gem install bundler && bundle update
+  #   - run: bundle exec rake circleci:build
+
+  # test-ruby-2.4:
+  #   docker:
+  #   - image: circleci/ruby:2.4.4-stretch
+  #   working_directory: ~/googleapis/google-cloud-ruby
+  #   shell: /bin/bash --login -eo pipefail
+  #   steps:
+  #   - checkout
+  #   - run:
+  #       name: Install bundle
+  #       command: gem update --system && gem install bundler && bundle update
+  #   - run: bundle exec rake circleci:build
+
+  # test-ruby-2.3:
+  #   docker:
+  #   - image: circleci/ruby:2.3.7-stretch
+  #   working_directory: ~/googleapis/google-cloud-ruby
+  #   shell: /bin/bash --login -eo pipefail
+  #   steps:
+  #   - checkout
+  #   - run:
+  #       name: Install bundle
+  #       command: gem update --system && gem install bundler && bundle update
+  #   - run: bundle exec rake circleci:build
+
+  # post:
+  #   docker:
+  #   - image: circleci/ruby:2.5.1-stretch
+  #   working_directory: ~/googleapis/google-cloud-ruby
+  #   shell: /bin/bash --login -eo pipefail
+  #   steps:
+  #   - checkout
+  #   - run:
+  #       name: Install bundle
+  #       command: gem update --system && gem install bundler && bundle update
+  #   - run: bundle exec rake circleci:post
+
   release:
```

Comment:
I want to leave circle's release as is. Kokoro is currently only meant to be taking over PR's, merge-to-master's, and docs build.

Reply:
Thanks, sgtm. Will the Kokoro build fail (meaning, report the error) if there is any error while building the docs?

Reply:
It will fail if any part of `rake circleci:post` returns an error. Why was circle's post job not failing? Should we put the contents of the Rakefile's :post block in a `begin rescue` with more explicit error handling to avoid silent failures?

Reply:
Sounds good, I'm guessing any error will fail the Kokoro build unless you know otherwise. I'm not even sure this is still an issue with Circle 2, just thought I'd ask.